src/interfaces/hybrid_crypto/statics.rs

```diff
@@ -23,6 +22,7 @@ where
 }
 
 impl<S: SymmetricCrypto> EncryptedHeader<S> {
+    #[allow(dead_code)]
     pub(crate) fn as_bytes(&self) -> anyhow::Result<Vec<u8>> {
```

Comment:
we can change visibility to just `pub` and remove `allow(dead_code)`

Reply:
done